core/src/main/java/org/infinispan/interceptors/VersionedEntryWrappingInterceptor.java

```diff
@@ -41,33 +41,47 @@ public void initialize(VersionGenerator versionGenerator) {
 
    @Override
    public Object visitPrepareCommand(TxInvocationContext ctx, PrepareCommand command) throws Throwable {
+      VersionedPrepareCommand versionedPrepareCommand = (VersionedPrepareCommand) command;
```

Comment:
To be honest it looks like the changes here are just cleaning up the class and nothing with this PR, right?